src/core/Browser.js

```diff
@@ -26,6 +26,8 @@
 	    chrome    = ua.indexOf('chrome') !== -1,
 	    gecko     = ua.indexOf('gecko') !== -1  && !webkit && !window.opera && !ie,
 
+	    win = navigator.platform.indexOf('Win') === 0,

```

Comment:
Yeah, detecting the OS is much more reliable if `navigator.platform` is used. I just really really hope there will be no weird combinations of Chrome in random WinPhones or WinTablets plus USB mice.